contracts/test/consensus/SpyCoConsensus.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+contract SpyCoConsensus {
+
+    mapping(bytes32 => address) public anchors;
+
+    bool public called;
+
+    function setAnchorAddress(bytes32 _metachainId, address anchor) public {
+        anchors[_metachainId] = anchor;
+    }
+
+    function getAnchor(bytes32 _metachainId) public returns(address) {
+        called = true;
```

Comment:
Instead of having `bool public called;` add `bytes32 spyMetachainId`.
In tests assert that `spyMetachainId` value is the same that was passed as the params while calling `getAnchor`

Reply:
Done